ruby/Gemfile.lock

```diff
@@ -34,4 +34,4 @@ DEPENDENCIES
   sinatra
 
 BUNDLED WITH
-   1.17.2
```

Comment:
This probably shouldn't be here in the java PR

Reply:
deleted